react/src/components/molecules/CalloutLink/CalloutLink.stories.js

```diff
@@ -8,7 +8,7 @@ import CalloutLink from './index';
 import CalloutLinkDocs from './CalloutLink.md';
 import calloutLinkOptions from './CalloutLink.knobs.options';
 
-storiesOf('molecules/CalloutLink', module).addDecorator(withKnobs)
```

Comment:
Was this move discussed? these were previously grouped as variations of the same molecule, now just bare in the molecules listing:
![screen shot 2018-09-13 at 16 17 48](https://user-images.githubusercontent.com/38226003/45518997-a5c26200-b770-11e8-8071-2ad991cffac9.png)